compiler/surface/lexer_fr.cppo.ml

```diff
@@ -34,6 +34,8 @@
 #define MS_OF "de"
 #define MS_LIST "liste de"
 #define MR_LIST "liste", space_plus, "de"
+#define MS_OPTION "Optionnel de"
```

Comment:
Not sure whether `Optional` should be capitalized here, as `liste` is not. I would be in favor of having those builtin polymorphic types to not be capitalized.

Reply:
I agree it's a problem, but was considering solving the discrepancy the other way round 😁 
the point being that `Optionnel` is still somehow a variant type and that's the convention for them...

but I guess built-in types are uncapitalised, and it may make as much sense to consider `list` and `optional` in that category vs. user-defined types which are capitalised...

What's your final takeaway ?